client/src/pages/Home.css

```
@@ -9,3 +9,46 @@
 .message:hover {
 	background-color: yellow;
 }
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
```

```suggestion
  flex-direction: column;
  gap: 1rem;
```
We can use the `gap` property to add space between all the child elements. This way, we don't have to add margin-top/margin-bottom to every child element to add space.